### DISCUSSION TITLE
Change default content of the list item

### DISCUSSION THREAD

**Discussion by piranhoid** [ueberdosis/tiptap#2103] [General Questions] [answered]
Hi me again 😅 , 

Now that i have override the default paragraph node, to add an add button, left to the block, i have a big issue

Indeed the default content of the list item it's the paragraph node but that i don't want to have that in my list item 

So i have changed the default content of the ListItem with 'text*' , so now i have a type text in my listItem 

But i have loose the shortcut '-' and the 'Enter' to leave the list item

Exemple : https://codesandbox.io/s/tiptap-gs-84njb?file=/src/Tiptap.tsx

**Comment by piranhoid** [comment, marked as answer]
Is it because bulletList is not on the same group as text ?

**Reply by cadars** [reply]
You might want to have a look at https://github.com/ueberdosis/tiptap/issues/118.

**Reply by piranhoid** [reply]
Well i think it will be complicated to change the default behavior of the list item, maybe try to not override the default paragraph, and create a custom node (with the add button in it) only on a new empty line ....


But ! 

If i want to add a drag button on my paragraph, i don't want to my list item content to be draggable 😢 

Any idea or a starting point ? 

I have seen a lot of implementation for exemple on Complish, on wich the add and drag button are not linked with the paragraph node

**Reply by hanspagel** [reply]
The extension that Complish uses will be released as a Tiptap Pro extension soon:

https://tiptap.dev/pricing

**Reply by piranhoid** [reply]
Thank your for your answer!

We are really liking TipTap and we find it really useful for our company needs and considering sponsoring/paying for it, but this feature is really important to us.

So if I may ask, do you have any kind of ETA for the release of this extension? If not, do you have any advice to implement

Thank you for your time

**Reply by hanspagel** [reply]
Cool! Not really an ETA, but it’s very much on the top of our priority list to get that out. I hope we’re able to release it in November. ✌️
